nltk/parse/dependencygraph.py

```diff
@@ -321,10 +326,15 @@ def extract_10_cells(cells):
                         'CoNLL(10) or Malt-Tab(4) format'.format(cell_number)
                     )
 
-            word, lemma, ctag, tag, feats, head, rel = cell_extractor(cells)
+            line_index, word, lemma, ctag, tag, feats, head, rel = cell_extractor(cells)
```

Comment:
I've wrapped this call in a try/except that should make sure the change is backwards compatible (and added a doctest to verify this). See if you agree or if you see any potential problems.